qa-tests-backend/src/test/groovy/VulnReportingTest.groovy

```diff
@@ -16,7 +16,7 @@ import spock.lang.IgnoreIf
 import util.Env
 
 // slack notifications are not supported on P/Z
-@IgnoreIf({ Env.REMOTE_CLUSTER_ARCH == "ppc64le" || Env.REMOTE_CLUSTER_ARCH == "s390x" })
+//@IgnoreIf({ Env.REMOTE_CLUSTER_ARCH == "ppc64le" || Env.REMOTE_CLUSTER_ARCH == "s390x" })
```

Comment:
Uncomment this since we don't have MA test image for this test & its expected to fail. Let this test be ignored for now on P/Z